packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -717,3 +719,9 @@ export interface GetIndexerVrtxTokenInfoParams {
  * For instance, 10 VRTX will be represented as 10.0, not 10*10^18.
  */
 export type GetIndexerVrtxTokenInfoResponse = number;
+
+export type GetIndexerFastWithdrawalSignatureParams =
+  IndexerServerFastWithdrawalSignatureParams;
+
+export type GetIndexerFastWithdrawalSignatureResponse =
```

Comment:
Might be a bit annoying but for clealiness, can we rename `tx_bytes` to camelcase when we convert from server -> client response?